tests/testthat/test-citation_authors.R

```diff
@@ -0,0 +1,13 @@
+context("build citation/authors")
```

Comment:
For new tests, I'd like pkgdown to move to our emerging standard - the name of the test file should be the same as the R file (but with `test-` prefixed), and the context should just be the file name. You can most easily achieve this with `usethis::use_test()`.  

If that results in test files that are too big (or too small) it's an indication we also need to refactor the `.R` files